ocs_ci/framework/conf/default_config.yaml

```diff
@@ -72,7 +72,7 @@ REPORTING:
 # --cluster-conf file.yaml data you will pass to the pytest.
 ENV_DATA:
   cluster_name: null  # will be changed in ocscilib plugin
-  storage_cluster_name: 'rook-ceph'
+  storage_cluster_name: 'openshift-storage'
```

Comment:
From the email from @nehaberry I see that she was pointing to the name:  ocs-storagecluster 
```
Setup created using ocs-operator in UI
$ oc get cephfilesystem -n openshift-storage
NAME                                ACTIVEMDS   AGE
ocs-storagecluster-cephfilesystem   1           84m
Setup created using ocs-operator from ocs-ci(CLI)
$ oc get cephfilesystem -n openshift-storage
NAME                       ACTIVEMDS   AGE
rook-ceph-cephfilesystem   1           5h58m
```

Reply:
Fixed